manifest.webapp

```diff
@@ -53,6 +53,10 @@
       "description": "Required by the cozy-bar to display Claudy and know which applications are coming soon",
       "type": "io.cozy.settings",
       "verbs": ["GET"]
+    },
+    "triggers": {
+      "type": "io.cozy.triggers",
+      "verbs": ["GET"]
```

Comment:
Should we add a description here @poupotte ?

Reply:
Used to sort contacts / Utilisé pour trier les contacts